docs/development/releasing.rst

```diff
@@ -175,26 +171,28 @@ packages that use the full bugfix/maintenance branch approach.)
 
 #. Push up all these changes to the `astropy core repository`_::
 
-      $ git push --tags upstream v<version>
+      $ git push --tags upstream v<version branch>.x
 
    .. note::
 
       You may need to replace ``upstream`` here with ``astropy`` or
       whatever remote name you use for the `astropy core repository`_.
 
-#. If this is a release of the current release (not an LTS), update the
-   "stable" branch to point to the new release::
+#. If this is a release of the current release (i.e., not an LTS supported along
+   side a more recent version), update the "stable" branch to point to the new
+   release::
 
       $ git checkout stable
       $ git reset --hard v<version>
       $ git push upstream stable --force
```

Comment:
Ohh, that bloody ``stable`` tag (for 0.2.1) keeps coming back no matter how many times I delete it.